src/index.html

```diff
@@ -1,13 +1,58 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="product-card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          class="product-card__img"
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <a href="#" class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+
+        <div class="product-card__code code">
+          <span class="code__title">Product code:</span>
+          <span class="code__value">195434</span>
+        </div>
+
+        <div class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+          </div>
+
+          <div class="product-card__reviews reviews">
+            <span class="reviews__title">Reviews:</span>
+            <span class="reviews__value">5</span>
+          </div>
+        </div>
+
+        <div class="product-card__price price">
+          <div class="price__title">Price:</div>
+          <div class="price__value">$2,199</div>
+        </div>
```

Comment:
2 BEM blocks will be enough, price is part of card, I don`t think that you will use somewhere price without card so it doesn`t make sense to create separate BEM block
![image](https://github.com/mate-academy/layout_product-cards/assets/64101898/93a9b364-d8ec-4fb3-a939-9ae18674205f)